src/app/admin/factories/factory.go

```diff
@@ -0,0 +1,15 @@
+package admin_factories
+
+import (
+	admin_handlers "clinic-api/src/app/admin/handlers"

```

Comment:
why not using camelcase?

Reply:
Because it's package name. I think it would be better if using snake case instead of using camel case.

Reply:
okay